core/hw/mem/_vmem.cpp

```diff
@@ -547,15 +547,30 @@ int ashmem_create_region(const char *name, size_t size)
 	void* _nvmem_alloc_mem()
 	{
 #ifndef _ANDROID
+#ifdef USE_VMEM_FILE
+		//shm is borken on the rapi
+		//using a file is slower, but works
+		//file in tmpfs would be better?
+		fd = -1;
+#else
 		fd = shm_open("/dcnzorz_mem", O_CREAT | O_EXCL | O_RDWR,S_IREAD | S_IWRITE);
 		shm_unlink("/dcnzorz_mem");
+#endif
 		if (fd==-1)
 		{
+#ifdef USE_VMEM_FILE
+			printf("Falling back to file allocation..\n");
```

Comment:
true, not needed, just wanted to "revert" the changes that were done explicitly for the RPi2 so no one complains about it, but yes it's actually not needed.